docs/design/flow-traffic-mirroring/flow-traffic-mirroring.md

```diff
@@ -0,0 +1,171 @@
+# Flow traffic mirroring
+
+## Idea
+There must be an ability to mirror traffic on a given flow and send it to a specific switch+port+vlan.
```

Comment:
Propose to replace `switch+port+vlan` with network endpoint, because even now it is incorrect (i.e. there should be 2nd vlan). In future our endpoint data set can mutate even more.

Reply:
After discussion, it was decided to use the 2-vlan approach, which is already used in the kilda system.